config/environments/test.rb

```diff
@@ -7,6 +7,11 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  config.middleware.use Clearance::BackDoor do |username|
+    Clearance.configuration.user_model.find_by(username: username)
+  end
```

Comment:
This isn't going to work because the `User `model doesn't have a `username` field. The default behavior should work just fine.
```suggestion
  config.middleware.use Clearance::BackDoor

```